lib/plugins/TaskRunner/Drupal.js

```diff
@@ -212,6 +224,26 @@ class Drupal extends LAMPApp {
     }
   }
 
+  addScriptDevelGenerate() {
+    this.script.push('cd $SRC_DIR ; drush en devel -y; drush en devel_generate -y;')
+  }
+
+  addScriptDevelGenerateContent(types) {
+    for (var content_type in types) {
+      if (types.hasOwnProperty(content_type)) {
+        this.script.push(`cd $SRC_DIR ; drush generate-content ${types[content_type]} --types=${content_type}`);
+      }
+    }
+  }
+
+  addScriptDevelGenerateUsers(roles) {
+    for (var role_name in roles) {
+      if (roles.hasOwnProperty(role_name)) {
```

Comment:
I always feel like something if amiss if you have to check if a key exists this way. Better would be to use _.keys (https://lodash.com/docs#keys) to create an array of keys and then use those to create the loop. I also don't feel great about the particular structure but it works. I think you'll also want to make sure the value is a number, currently you have no check for that.